cmd/juju/commands/main_test.go

```diff
@@ -236,6 +231,7 @@ var commandNames = []string{
 	"expose",
 	"generate-config", // alias for init
 	"get-config",
+	"get-configs",
```

Comment:
Our current direction is "all commands have singular and plural forms"; I would like to leave this until we have consensus next week.  It is easy enough to update later.